sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -978,6 +978,119 @@
           "id": "ew6b0sd",
           "dataGridLabel": false,
           "shortcutButtons": []
+        },

```

Comment:
![Screen Shot 2021-08-24 at 11 31 19 AM](https://user-images.githubusercontent.com/62901416/130670729-775a12f8-81da-4e34-abc1-4cd593c02909.png)
 Please check this "Allow Only available Values" . SO that only the type validation happens at form side also

Reply:
Sure. It has been modified in dev forms and form-definition updated too.